hyperspace/primitives/src/lib.rs

```diff
@@ -370,10 +370,25 @@ pub trait MisbehaviourHandler {
 	) -> Result<(), anyhow::Error>;
 }
 
+/// Provides an interface for syncing light clients to the latest state
+#[async_trait::async_trait]
```

Comment:
Can we remove this dependency and add the feature instead?
https://blog.rust-lang.org/inside-rust/2022/11/17/async-fn-in-trait-nightly.html

Reply:
Okay, we'll have to enforce nightly usage in the repo then

Reply:
@vmarkushin any ideas on this? I’m neutral

Reply:
@blasrodri are you sure that we should use an unstable feature instead of a stable crate? :)

Reply:
I wouldn't use nightly until it's really needed. In this case, I don't see a neeed for this

Reply:
Yeah let's keep it until it's in stable